components/posts/createPost.css

```diff
@@ -0,0 +1,8 @@
+.react-select__menu {
```

Comment:
`components/search/searchbar.css` is all lower case, where as this is `createPost.css`. 

Additionally, the directory `components/search` should be lower case to be uniform